README.md

```diff
@@ -18,6 +18,7 @@ To contribute:
 
 | Name  |  Location |  Application Period |  Notes |
 |---|---|---|---|
+|  [Blend](https://jobs.lever.co/blendlabs/2a469512-a8c2-44fa-a260-ef3ae0c90db7?lever-via=2F9sFPC6-M) | San Francisco, CA | Present |
```

Comment:
Add to the bottom of the list, and add expected graduation date of 2021 (per the page) to the notes section.